apps/vercel/frontend/src/hooks/useError/useError.tsx

```diff
@@ -0,0 +1,51 @@
+import { errorMessages } from '@constants/errorMessages';
+import { ConfigPageContext } from '@contexts/ConfigPageProvider';
+import { Errors, PreviewPathError } from '@customTypes/configPage';
+import { isEmpty, pickBy } from 'lodash';
+import { useContext } from 'react';
+
+type Error =
+  | Errors['authentication']
+  | Errors['apiPathSelection']
+  | Errors['projectSelection']
+  | PreviewPathError;
+type ErrorKeys =
+  | keyof Errors['authentication']
+  | keyof Errors['apiPathSelection']
+  | keyof Errors['projectSelection']
+  | keyof Omit<PreviewPathError, 'contentType'>;
+
+export const getErrorMessage = (error: Error, contentType?: string) => {
+  const currentError = pickBy(error);
+  let errorKey: ErrorKeys;
+
+  if (contentType) {
+    errorKey = Object.keys(currentError)[1] as ErrorKeys;
```

Comment:
pretty minor but could this whole block be a bit cleaner with a ternary operator instead of the mutable variable and if clause?

Reply:
sure! Have adjusted this 👍